spec/models/role_spec.rb

```diff
@@ -22,6 +23,20 @@
         expect(Role.admins).to match_array([admin_role])
       end
     end
+
+    describe ".moderators" do
+      it "should include admins" do
+        expect(Role.moderators).to include(admin_role)
+      end
+
+      it "should include moderators" do
+        expect(Role.moderators).to include(moderator_role)
+      end
+
+      it "should not include normal users" do
+        expect(Role.moderators).to_not include(person)
+      end
+    end
   end
```

Comment:
No. I'm saying that these tests are for the scope `moderators` and it's good to have them, but I'd also like to see them for `Role.moderator?(person)`, like we have specs for `Role.is_admin?(person)` below.